docker-compose-github-actions.yml

```diff
@@ -1,9 +1,35 @@
 version: '2.2'
 
 services:
+  mysql:
+    image: public.ecr.aws/unocha/mysql:10.6
+    hostname: mysql
+    container_name: hxl-proxy-mysql
+    environment:
+      MYSQL_DB: hxl_proxy
+      MYSQL_USER: hxl_proxy
+      MYSQL_PASS: hxl_proxy
+
+  redis:
+    image: public.ecr.aws/unocha/redis:5
+    hostname: redis
+    container_name: hxl-proxy-redis
```

Comment:
```suggestion

```
you don't need redis, didn't found a good use for it so far. Unless, of course, there are redis tests